docs/PROJECT_SETUP.md

```diff
@@ -23,9 +23,10 @@ AtomicCache::DefaultConfig.configure do |config|
   config.metrics = Datadog::Statsd.new('localhost', 8125, namespace: 'cache.atomic')
 
   # note: these values can also be set in an env file for env-specific settings
-  config.namespace      = 'atom'
-  config.cache_storage  = AtomicCache::Storage::SharedMemory.new
-  config.key_storage    = AtomicCache::Storage::SharedMemory.new
+  config.namespace        = 'atom'
+  config.default_options  = { generate_ttl_ms: 500 }
+  config.cache_storage    = AtomicCache::Storage::SharedMemory.new
+  config.key_storage      = AtomicCache::Storage::SharedMemory.new
```

Comment:
Being new to gem dev... where's the line in the sand for documentation? Should consumers in the RoR community go to the spec files to see examples for how some of the options are use? 

Would it be for the gem to provide an example for how a Rails app may have `config.dalli_store/memcache_store` options that would be passed into a dalli client or memory instance?

Similarly - is there value in showing an example of a `atomic_cache.fetch` call with options overriding defaults?

Reply:
I dont mean to say our way is THE way but eg:
the note above  `note: these values can also be set in an env file for env-specific settings`
```
  if Rails.env.development? || Rails.env.test?
    config.cache_storage  = AtomicCache::Storage::SharedMemory.new
    config.key_storage    = AtomicCache::Storage::SharedMemory.new
  elsif Rails.env.staging? || Rails.env.production?
    memcache_hosts = Ibotta::Application.config.memcache_hosts
    options = Ibotta::Application.config.cache_store_options

    dc = Dalli::Client.new(memcache_hosts, options)
    config.cache_storage  = AtomicCache::Storage::Dalli.new(dc)
    config.key_storage    = AtomicCache::Storage::Dalli.new(dc)
  end
```

Reply:
good question. You could have a specific rails example for this maybe (leaving out the Ibotta prefixes ;) )

Reply:
Being an open source project, this is something I've struggled a bit with, is to find a level of documentation that's helpful but not overly Ibotta-specific. I'll try and roll what you have up there into something this morning, and see if we can find a happy middle ground.

Reply:
Added a `More Complex Rails Configuration` section down below. Have a look. Resolving this conversation.